SPECS/python-jinja2/python-jinja2.spec

```diff
@@ -1,13 +1,13 @@
 Summary:        A fast and easy to use template engine written in pure Python
 Name:           python-jinja2
-Version:        2.10.1
-Release:        4%{?dist}
+Version:        3.0.3
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages/Python
 URL:            https://jinja.pocoo.org/
-Source0:        https://files.pythonhosted.org/packages/93/ea/d884a06f8c7f9b7afbc8138b762e80479fb17aedbbe2b06515a12de9378d/Jinja2-%{version}.tar.gz
+Source0:        https://files.pythonhosted.org/packages/91/a5/429efc6246119e1e3fbf562c00187d04e83e54619249eb732bb423efa6c6/Jinja2-%{version}.tar.gz
```

Comment:
Nit: We can use a nicer URL here that allows for easier upgrades.

https://pypi.io/packages/source/J/Jinja2/Jinja2-3.0.3.tar.gz